Emby.Server.Implementations/LiveTv/TunerHosts/HdHomerun/HdHomerunManager.cs

```diff
@@ -78,73 +79,75 @@ public HdHomerunChannelCommands(string channel, string profile)
 
     public class HdHomerunManager : IDisposable
     {
-        public static int HdHomeRunPort = 65001;
+        public const int HdHomeRunPort = 65001;
         // Message constants
-        private static byte GetSetName = 3;
-        private static byte GetSetValue = 4;
-        private static byte GetSetLockkey = 21;
-        private static ushort GetSetRequest = 4;
-        private static ushort GetSetReply = 5;
+        private const byte GetSetName = 3;
+        private const byte GetSetValue = 4;
+        private const byte GetSetLockkey = 21;
+        private const ushort GetSetRequest = 4;
+        private const ushort GetSetReply = 5;
+
+        private readonly ILogger _logger;
 
         private uint? _lockkey = null;
         private int _activeTuner = -1;
-        private readonly ISocketFactory _socketFactory;
-        private IPAddress _remoteIp;
+        private IPEndPoint _remoteEndPoint;
 
-        private ILogger _logger;
-        private ISocket _currentTcpSocket;
+        private TcpClient _tcpClient;
 
-        public HdHomerunManager(ISocketFactory socketFactory, ILogger logger)
+        public HdHomerunManager(ILogger logger)
         {
-            _socketFactory = socketFactory;
             _logger = logger;
         }
 
         public void Dispose()
         {
-            using (var socket = _currentTcpSocket)
+            using (var socket = _tcpClient)
             {
                 if (socket != null)
                 {
-                    _currentTcpSocket = null;
+                    _tcpClient = null;
 
-                    var task = StopStreaming(socket);
+                    var task = StopStreaming(_tcpClient);
```

Comment:
Are you not calling StopStreaming with null here? Can we put null as a constant here?

Reply:
Fixed it